### DISCUSSION TITLE
ModuleNotFoundError: No module named 'earthpy.appeears'

### DISCUSSION THREAD

**Discussion by annieM459** [earthlab-education/Earth-Analytics-2023-01-Intro#373] [Q&A04 Raster Data]
I decided to start working on my own boundary and created a new jupyter notebook in my website github. However, I keep receiving the below error. I repasted in the .json container incase something was up with that but am still running in to this issue. 
<img width="613" alt="Screenshot 2023-09-25 at 3 14 13 PM" src="https://github.com/earthlab-education/Earth-Analytics-2023-01-Intro/assets/140002702/696f04dc-10d8-4381-b51f-75977cac0dfd">

**Comment by JKinsler** [comment]
You have to pip install appeears as a separate module when working locally

`!pip3 install appeears`

then you can import it:
`import appeears as etapp`

**Reply by annieM459** [reply]
<img width="494" alt="Screenshot 2023-09-26 at 10 48 50 AM" src="https://github.com/earthlab-education/Earth-Analytics-2023-01-Intro/assets/140002702/b95fc2f7-da32-46fa-a760-8ed41255e376">

Bummer, I tried that and this happened!

**Reply by annieM459** [reply]
Actually! I figured it out, that code was meant for a code cell. It ran! Thank you.

**Comment by earthlab-test-student** [comment]
`appeears` is a different package than we are using. The best way to get the `earthpy` package which contains the `earthpy.appears` module is to delete your Codespace and open a new one - there were updates after some folks started their Codespaces.
